www/src/components/plugin-searchbar-body.js

```diff
@@ -467,17 +467,16 @@ class PluginSearchBar extends Component {
 
   urlToSearch = () => {
     if (this.props.location.search) {
-      // ignore this automatically added query parameter
-      const search = this.props.location.search
-        .replace(`no-cache=1`, ``)
-        .slice(2)
+      const params = new URLSearchParams(this.props.location.search)
```

Comment:
Not sure if it's relevant to us, but `URLSearchParams` is not supported by IE11 ( https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Browser_compatibility )

Reply:
there is a polyfill https://github.com/ungap/url-search-params